include/superior_mysqlpp/dnsa_connection_pool.hpp

```diff
@@ -39,6 +41,13 @@ namespace SuperiorMySqlpp
             DnsResolver& operator=(const DnsResolver&) = default;
             DnsResolver& operator=(DnsResolver&&) = default;
 
+            /**
+             * Resolve a hostname to ip addresses.
+             *
+             * @tparam const reference to std::string
+             * @param hostname
+             * @return Vector of strings that represent ip addresses for given hostname.
+             */
```

Comment:
I'm wondering if it wouldn't be nice whether methods docs should have throws section. 
At least here, IMHO it makes sense to mention possible asio exceptions.

Reply:
What do you think?
Also @OpatrilPeter @smradCZ